operations/export.go

```diff
@@ -1,151 +0,0 @@
-package operations
-
-import (
-	"context"
-	"io"
-
-	"github.com/evergreen-ci/evergreen/util"
-	"github.com/pkg/errors"
-	"github.com/urfave/cli"
-)
-
-const (
-	granularityHours   = "hour"
-	granularityMinutes = "minute"
-	granularitySeconds = "second"
-	granularityDays    = "day"
-)
-
-func Export() cli.Command {
```

Comment:
why are we deleting this?

Reply:
(we should probably delete the code that it calls on the server or hook it back up?)

Reply:
If there is a server endpoint for this, we should check to see if anyone has called it in the last 90 days.

Reply:
There have been a few dozen calls to the `/scheduler/*` endpoints called by this function in the past 90 days.
It seems like [this PR](https://github.com/evergreen-ci/evergreen/pull/481) refactored the CLI commands but didn't add `Export()` back to the CLI. Instead of removing it, I could just re-add it to the CLI.